tests/integration/test_mysql_reader.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from nck.readers.sql_reader import SQLReader
+
+class TestMySQlReader:
+
+        basic_query = "SELECT * FROM employees"
+
+        # engine = SQLReader._create_engine(
+        #     "127.0.0.1",
+        #     "3306",
+        #     "test",
+        #     "secret",
+        #     "employees"
+        # )
+
+        engine = SQLReader(
```

Comment:
Move all configurations to environment variables or to an external python config file for less variables dependencies !

Reply:
Especially if you already declared the values as environment variables to the mysql container in the docker-compose file ([declare it in a .env file and use the env_file parameter](https://medium.com/better-programming/using-variables-in-docker-compose-265a604c2006#:~:text=If%20we%20declare%20the%20same,declared%20in%20the%20Compose%20file.) in both container declarations instead)